sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,4 @@
+SET @GUID := NULL;
+DELETE FROM `gameobject` WHERE guid = @GUID;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneid`, `areaid`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES
+(@GUID,149431,109,0,0,1,1,-518.154,-85.2353,-74.488,3.14159,0,0,0,0,43200,100,1);
```

Comment:
```sql
--
SET @GUID := 11011; -- (unused GUID from TDB 335.24011)

-- spawn forcefield (149431)
DELETE FROM `gameobject` WHERE `guid` = @GUID;
INSERT INTO `gameobject` (`guid`,`id`,`map`,`zoneId`,`areaId`,`spawnMask`,`phaseMask`,`position_x`,`position_y`,`position_z`,`orientation`,`rotation0`,`rotation1`,`rotation2`,`rotation3`,`spawntimesecs`,`animprogress`,`state`,`ScriptName`,`VerifiedBuild`) VALUES
(@GUID, 149431, 109, 0, 0, 1, 1, -518.154, -85.2353, -74.488, 3.14159, 0, 0, 0, 0, 43200, 100, 1, '', 0);
```
if you don't know a free GUID from DB, you could choose a number higher than the last entry in the table `gameobject` and just comment that actual GUID should be selected by the TC team.

Not sure if the end marking in your SQL is a missing CR/LF, but since there is no line number, I suspect it is. See if you can manage to update the SQL and have a natural Line Feed or NewLine after the last semicolon.

By the way: I used all 20 column names, including `ScriptName` and `VerifiedBuild` (recommended codestyle).

My unused GUID was picked from TDB 335.24011 -- TC rev. b4783d6cace4

Reply:
Note that my suggested update to the SQL contains Zone: 1477 (The Temple of Atal'Hakkar), Area: 1477 (The Temple of Atal'Hakkar) (`zoneId`, `areaId` = 1477). Do you know if the `forcefield` needs 0 instead?

Reply:
Thanks for the tips.
I specified `NULL` because I saw in one of the pull requests that for an identical situation where they added something to the `gameobject` table they specified `NULL`. I will use your method from now on.

As for `zoneId` and `areaId`, as I know these columns are filled in by the **worldserver**, but I'm not sure.

I will also correct your other comments when we clarify the situation with `zoneId` and `areaId`.

Reply:
there is no need to fill zoneid or areaid.

Reply:
> there is no need to fill zoneid or areaid.

Thank you. Good point.